doc/whats_new.rst

```diff
@@ -263,6 +263,10 @@ Enhancements
      generating attribute ``estimators_samples_`` only when it is needed.
      By `David Staub`_.
 
+   - Added `labels` flag to :class:`metrics.log_loss` to correct metrics when
+     only one class is present in test data set.
```

Comment:
WDYT about rewording this to (or something similar)

"Added labels flag to :class:`metrics.log_loss` to explicitly provide the labels when the number of classes in y_true and y_pred differ"